md/profileCreation.md

```diff
@@ -0,0 +1,79 @@
+
+There is a special file of profiles for default profiles: `default_profiles.xml`  
+This file can only contain default profiles and cannot be renamed or deleted.
+
+Only Performance and Efficiency editions can create new file of profiles, in order to create custom profiles.
+
+## Create and build a profile
+
+::: info
+**Note:** All Bonita editions come with an .xml editor for this feature. Only Performance and Efficiency editions also come with a graphical UI.
+:::
+
+From Bonita Studio, click on **Organization** -> **Profile** -> **New...**  
+It will create an empty file of profiles with a default name and open an editor. You can **rename** this file from the graphical UI or from the menu **Open**.  
+
+The profile graphical UI provided is bound to the xml source, so you can use it to build your profile without writing xml.  
+You can switch from the graphical UI to the xml source at any moment, using the tabs _Editor_ and _Source_ at the bottom of the editor.  
+A toolbar is embedded to the editor, use it to:  
+
+ - **Add** a profile to this file of profiles: you can add a new profile or duplicate an existing one from your current repository
+ - **Save** this file of profiles **As** an other file of profiles (useful to duplicate the whole file)
+ - **Deploy** this file of profiles (i.e all the profiles in this file)
+ ⚠  For _development_ purposes, it also deploys their referenced pages used as portal menu entries, but if those resources use REST API Extensions, you have to deploy those manually. In _production_ environment, you still need to import pages, and REST API Extensions manually **before** to import the file of profiles.   
+ - **Rename** this file of profiles
+ - **Export** this file of profiles (download the .xml)
+ - **Delete** this file of profiles from your current workspace
+ - **Manage** the Organization: add users / groups / roles to the organization in order to map them to this profile
+ - **Restore** the default file of profiles to its initial state (_only available for the file_ `default_profile.xml`) 
+
+Using the graphical editor, you can set all the parameters of your profile.
+
+The **profile name** has to be unique in a given file of profiles, else it will lead to conflicts when you will try to deploy the file of profiles.  
+You can duplicate a profile name in different files, but remember that you can only have one instance of a given profile in production.
+
+In the mapping section you can map Users / groups / roles / memberships to your profile. You have to map your profile with elements of the active organization.  
+⚠  Nodes in the group tree are **independent** ⚠  It means that mapping `/acme` to your profile won't map `/acme/hr` to it. Each node has to be mapped individually to be took into account. 
```

Comment:
what's the difference? The mapping doesn't change, it's just that you don't have the tool in prod, but we are talking about the studio feature here no?